### DISCUSSION TITLE
What framework did you use to create the Argo CD documentation

### DISCUSSION THREAD

**Discussion by reselbob** [argoproj/argo-cd#7027] [Q&A] [answered]
[https://argoproj.github.io/argo-cd/](https://argoproj.github.io/argo-cd/)

I am interested to use the documentation framework for one of my projects.

Thanks in advance for your help.

**Comment by jannfis** [comment, marked as answer]
Hey @reselbob, we use [MkDocs](https://www.mkdocs.org/) with the [Material theme](https://squidfunk.github.io/mkdocs-material/) for rendering the docs.

You can look at our configuration [here](https://github.com/argoproj/argo-cd/blob/master/mkdocs.yml)

HTH.

**Comment by reselbob** [comment]
Thanks!
